app/apps/server/communication/rest.js

```diff
@@ -658,12 +693,72 @@ export class AppsRestApi {
 				}
 			},
 			post() {
-				if (!this.bodyParams.setting) {
-					return API.v1.failure('Setting to update to must be present on the posted body.');
+				try {
+					Promise.await(updateAppSettingById(this.urlParams.id, this.urlParams.settingId, this.bodyParams.value));
+					return API.v1.success();
```

Comment:
With this change in `[POST]/{id}/settings/{setting-id}` We will only be able to update the value of the setting. Earlier RC supported other properties as well. Can we also support other properties?

Reply:
This endpoint can now update any setting property.